tests/test_google_sheets_start_date.py

```diff
@@ -0,0 +1,162 @@
+import os
```

Comment:
Please remove this test. The start date is not a feature used by the tap. There is a separate ticket to remove it from the configurable properties. https://jira.talendforge.org/browse/TDL-14262

Reply:
Removed it.